spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/MetricsAutoConfiguration.java

```diff
@@ -75,6 +78,11 @@
 		SimpleExportConfiguration.class, StatsdExportConfiguration.class })
 @AutoConfigureAfter(DataSourceAutoConfiguration.class)
 public class MetricsAutoConfiguration {
+	@Bean
+	@Order(0)
```

Comment:
Sanity check please?

Reply:
I don't think you can use `@Order` on `@Bean` methods.

Reply:
Scrub that, yes you can.